lib/factory-helper/date.rb

```diff
@@ -3,7 +3,7 @@ class Date < Base
 
     class << self
 
-      def between(from, to)
+      def between(from=::Date.today-10, to=::Date.today+10)
```

Comment:
I didn't give much thought to the default values, here and elsewhere, just put something in to make tests pass. Input welcome.